lib/extension-core.js

```diff
@@ -150,4 +150,26 @@ function core(loader) {
     }
     return loaderInstantiate.call(loader, load);
   }
+
+  function applyExtensions(extensions, loader) {
+    loader._extensions = [];
+    var ext;
+    for(var i = 0, len = extensions.length; i < len; i++) {
+      ext = extensions[i];
+      ext(loader);
+      loader._extensions.push(ext);
+    }
+  }
+
+  loader.clone = function() {
+    var originalLoader = this;
+    var loader = new LoaderPolyfill(originalSystem);
+    loader.baseURL = originalLoader.baseURL;
+    loader.paths = { '*': '*.js' };
+    loader.originalLoader = originalLoader;
+    applyExtensions(originalLoader._extensions, loader);
+    addTraceurPaths(loader);
```

Comment:
Yes please do. Will merge to master as soon as 6to5 3 is released in the next few hours.